server/constants/platform.ts

```diff
@@ -4,16 +4,20 @@ import moment from 'moment';
 import { SupportedCurrency } from './currencies';
 
 const isProdOrStaging = config.env === 'production' || config.env === 'staging';
-const MIGRATION_DATE = moment('2024-10-01T00:00:00Z');
+
+/**
+ * Defines the transition date and time from OCI to Ofico.
+ */
+export const PLATFORM_MIGRATION_DATE = moment('2024-10-01T00:00:00Z');
 
 /**
  * Returns the platform constants based on a function to check if the platform has been migrated.
  */
 const getPlatformConstants = (checkIfMigrated: () => boolean) => ({
   get PlatformCollectiveId() {
     if (checkIfMigrated()) {
-      // ofico
-      return 835523;
+      // OFT - Open Finance Technologies
+      return 845576;
```

Comment:
@hdiniz Changed to Ofitech based on https://opencollective.slack.com/archives/C06K5J91675/p1727428802761569?thread_ts=1727275801.528919&cid=C06K5J91675